rally-domain/src/main/groovy/yakworks/rally/activity/model/Task.groovy

```diff
@@ -56,27 +56,18 @@ class Task implements RepoEntity<Task>, Serializable {
     }
 
     static constraintsMap = [
-        dueDate:[ description: 'The relative path to the locationKey',
-            nullable: false],
-        state:[ description: 'Defaults to Open',
-            nullable: false, required: false],
-        status:[ description: 'Defaults to TaskStatus.OPEN',
-            nullable: false, required: false],
-        taskType:[ description: 'The type of the task',
-            nullable: false],
+        dueDate:[description: 'The relative path to the locationKey', nullable: false],
```

Comment:
if we are going to update all these and show the change then lets get the shortened `d` in place too instead of description.